packages/fxa-settings/src/pages/InlineRecoverySetup/mocks.ts

```diff
@@ -0,0 +1,12 @@
+export const RECOVERY_CODES = [
+  'A1B2',
+  'C3D4',
+  'E5F7',
+  'G8H9',
+  'I0J1',
+  'K2L3',
+  'M4N5',
+  '6O7P',
+];
+
+export const SERVICE_NAME = 'Example Service';
```

Comment:
Feels like we should at some point be creating a more global `mocks.tsx` file instead of having to redefine `MOCK_SERVICE_NAME` and others repeatedly, hmm. I wouldn't mind it living in `fxa-settings/src/mocks.ts` personally with a note at the top.

Doesn't have to be done in this PR btw but I think we'll want that sometime.

Reply:
I'm down for that!